### PR TITLE
[WIP]Support get chunkId for composite blob

### PR DESCRIPTION
**Pull request by SophieGuo410**

### CODE REVIEW

File: ambry-tools/src/main/java/com/github/ambry/tools/admin/GetChunkIdTool.java [hunk 1]
```diff
@@ -0,0 +1,243 @@
+/**
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
```
Comment (ankagrawal) [inline ambry-tools/src/main/java/com/github/ambry/tools/admin/GetChunkIdTool.java:2]: 2022

**Comment by cgtz** [comment]
I would suggest making this into an "admin" subresource similar to the "Replicas" frontend api instead of a CLI tool. I remember that the CLI tool was a bit hard to get running in an urgent situation, since it requires supplying a lot of configuration.

**Comment by SophieGuo410** [comment]
> 

yeah, I'll work on it. This is a temporarily work around for this use case.

**Comment by snalli** [comment]
does this need any review or can this be closed ?

**Comment by SophieGuo410** [comment]
> does this need any review or can this be closed ?

it's a draft pr and it's still work in progress.  no need to review for now.

**Pull request closed without merging**